picard/ui/itemviews.py

```diff
@@ -245,9 +246,9 @@ def remove(self, objects):
 class BaseTreeView(QtWidgets.QTreeWidget):
 
     options = [
-        config.Option("setting", "color_saved", QtGui.QColor(0, 128, 0)),
-        config.Option("setting", "color_error", QtGui.QColor(200, 0, 0)),
-        config.Option("setting", "color_pending", QtGui.QColor(128, 128, 128)),
+        #config.Option("setting", "color_saved", QtGui.QColor(0, 128, 0)),
+        #config.Option("setting", "color_error", QtGui.QColor(200, 0, 0)),
+        #config.Option("setting", "color_pending", QtGui.QColor(128, 128, 128)),
```

Comment:
I would suggest to completely remove those lines here

Reply:
It's planned.